### ISSUE TITLE
Active/Inactive label does not match current user status

### ISSUE THREAD

**Issue by obamarodrigue** [ignacionelson/ProjectSend#133]
Hi everybody!
When you Deactivate a user, the status does not change, it remains green active label. How can i fix that?!

And I also have a couple of suggestions...
I installed ProjectSend on a CentOS 6.5 server. And a friend asked me if it's not possible to add the previex for other files formats (PDF, Word or Excel).
Am learning PHP develpment... So if someone else more advanced than me can give it a try.... I think it will be cool fr milins of people around the world using ProjectSend.
Thanks!!!!

**Comment by lenamtl** [comment]
There are some user that are using CentOS you can search from the issue list.
I cannot reproduce the problem, please update to the latest PS version if possible
and check js error Chrome  right click select Inspect

**Comment by ASMoreira** [comment]
I also have the same problem.
Just changed === to == in clients.php line 324 and 325

**Comment by lenamtl** [comment]
I got the problem online today it's appear online in on users.php and clients.php

clients.php
```
/**
 * 4- Get active status
 */
	$status_hidden	= __('Inactive','cftp_admin');
	$status_visible	= __('Active','cftp_admin');
	$label			= ($row['active'] == 0) ? $status_hidden : $status_visible;
	$class			= ($row['active'] == 0) ? 'danger' : 'success';
```
users.php
```
/**
* 2- Get active status
 */
$status_hidden	= __('Inactive','cftp_admin');
$status_visible	= __('Active','cftp_admin');
$label			= ($row['active'] == 0) ? $status_hidden : $status_visible;
$class			= ($row['active'] == 0) ? 'danger' : 'success';
```

**Comment by ignacionelson** [comment]
Fixed on b175f35
